src/index.html

```diff
@@ -22,6 +22,375 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="https://imgbb.com/"
+        class="logo"
+      >
```

Comment:
Fix it everywhere if you have only 2 attributes
```suggestion
      <a href="https://imgbb.com/" class="logo">
```